docs/sidebars.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
             items: ["venv-setup", "intellij-setup"],
             label: "Setup Guide"
         },
+        "sdk-overview",
```

Comment:
Should this go before the setup guide?

Reply:
Moved to before setup guide, since users will want to see an overview of what the product is about first.